src/essential_validator.py

```diff
@@ -252,6 +252,11 @@ def download_file(self, file_info):
         # todo set download file 
         download_file = os.path.join(S3_DOWNLOAD_DIR, file_info[FILE_NAME])
         try:
+            if not self.bucket.file_exists_on_s3(key):
```

Comment:
I thought we've merged change in another PR, and created a constant for the message, please double check this.

Reply:
Updated in 2.0.0,  but not in 3.1.0.  Need merge changes in 2.0.0 to 3.1.0

Reply:
Updated by merge the fixes in 2.0.0 to 3.1.0